lib/registries/addon/components/registries-navbar/template.hbs

```diff
@@ -62,24 +62,58 @@
                 <nav.links.secondary
                     data-test-my-reviewing-button
                     data-analytics-name='Go to moderation view'
-                    local-class='SecondaryNavLinkButton'
+                    local-class='SecondaryNavLinkButton HideOnMobile'
                     @route='registries.branded.moderation'
                     @models={{array this.provider.id}}
                 >
                     {{t 'navbar.moderation'}}
                 </nav.links.secondary>
+                <nav.links.secondary
+                    data-test-my-reviewing-button
+                    data-analytics-name='Go to moderation view'
+                    local-class='SecondaryNavLinkButton OnlyOnMobile'
```

Comment:
Needs an aria-label attribute as there is no "discernible text" to indicate where this link point to.